product_pack/models/product_template.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    pack_type = fields.Selection([
```

Comment:
Why is this configured at template level and the pack lines at variant level? This seems totally inconsistent

Reply:
Why could this be inconsistent?
I suppose product_type is in the template because in that field both the template and the attributes must have the same value, which seems fine to me.
On the other hand, it seems that there should be a list of pack lines for each product variant, that may be correct in that way, what do you think?